app/lib/api/file_system/file_system_remote.dart

```diff
@@ -144,9 +144,29 @@ mixin RemoteSystem {
 
   Future<void> clearCachedContent() async {
     var cacheDir = await getRemoteCacheDirectory();
+    print('Cache directory path: $cacheDir');
     var directory = Directory(cacheDir);
-    if (await directory.exists()) {
-      await directory.delete(recursive: true);
+    bool exists = await directory.exists();
+    print('Directory exists: $exists');
+    if (exists) {
+      try {
+        await directory.delete(recursive: true);
+        print('Directory deleted successfully.');
+      } on FileSystemException catch (e) {
+        if (e.osError?.errorCode == 32) {
+          print('Directory is in use, retrying... $e');
+          // Retry after a short delay
+          await Future.delayed(const Duration(seconds: 5));
+          await clearCachedContent();
```

Comment:
maybe a while with a limit is better?
Otherwise it can happen that you will get a stack overflow exception